fuel-client/src/client/schema/block.rs

```diff
@@ -120,6 +122,33 @@ pub struct Header {
     pub application_hash: Bytes32,
 }
 
+#[derive(cynic::InlineFragments, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub enum Consensus {
+    PoAConsensus(PoAConsensus),
+}
+
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub struct PoAConsensus {
+    pub signature: Signature,
+}
+
+impl Block {
+    /// Returns the block producer public key, if any.
+    pub fn block_producer(&self) -> Option<fuel_vm::fuel_crypto::PublicKey> {
```

Comment:
[nit] is there any reason for using fully qualified imports here instead importing at the top of the file then referencing the type?

Reply:
`PublicKey` is used only once in this function, importing it for the whole file is not practical. Plus, we don't have any types of `fuel-crypto`, so it will remain imports clean.